projects/admin-core/src/lib/application/application-form/application-form.component.html

```diff
@@ -0,0 +1,28 @@
+<form [formGroup]="applicationForm">
+  <mat-form-field>
+    <mat-label i18n>Name</mat-label>
+    <input matInput
+           tmAutoFocus
+           placeholder="Name"
+           i18n-placeholder
+           formControlName="name"
+    />
+  </mat-form-field>
+  <mat-form-field>
+    <mat-label i18n>Title</mat-label>
+    <input matInput placeholder="Title" i18n-placeholder formControlName="title" />
+  </mat-form-field>
+  <mat-form-field>
+    <mat-label i18n>Comments</mat-label>
+    <textarea formControlName="adminComments" matInput></textarea>
+  </mat-form-field>
+  <mat-form-field>
+    <mat-label i18n>Projection</mat-label>
+    <mat-select formControlName="crs" placeholder="Projection" i18n-placeholder>
+      <mat-option *ngFor="let projection of projections" [value]="projection.code">
```

Comment:
It would be nice to have the initial/max bounds update (to the extent of the projection) when choosing a projection.

Reply:
Good idea, will add that